src/govuk/components/hint/_index.scss

```diff
@@ -2,8 +2,6 @@
   .govuk-hint {
     @include govuk-font($size: 19);
 
-    display: block;
```

Comment:
I think we might want to keep this in for now – if we don't, users will need to make the change to their HTML markup (changing hints from spans to divs) which would then count as a breaking change – otherwise their hints will appear inline.

What do you think about leaving this in, but raising an issue to remove this later, and attach it to the 4.0 milestone?

Reply:
@36degrees How's this https://github.com/alphagov/govuk-frontend/issues/1856 ?